synapse/models/economic.py

```diff
@@ -134,6 +140,20 @@ def getModelDefs(self):
                         'doc': 'The econ:price of the purchase'}),
                 )),
 
+                ('econ:receipt:item', {}, (
+
+                    ('purchase', ('econ:purchase', {}), {
+                        'doc': 'The purchase that contains this line item.'}),
+
+                    ('count', ('int', {}), {
+                        'doc': 'The number of items included in this line item.'}),
+
+                    ('price', ('econ:price', {}), {
+                        'doc': 'The cost of this receipt line item.'}),
```

Comment:
Is this the value of a single line item, or the overall cost based on the count?

Reply:
The price of the line including the count as a multiplier. Do the docs need a tweak to make that clear?